daemon/src/peer/run_state.rs

```diff
@@ -138,6 +150,8 @@ pub enum Status {
     Online {
         /// Number of connected peers.
         connected: usize,
+        /// Connected peers
+        connected_peers: HashMap<PeerId, Vec<SocketAddr>>,
```

Comment:
Are the values of the map used somewhere? Or is this in preparation for another change which surfaces this for the up.

Reply:
This map is used to provide the data for the "Peers" panel in the network diagnostics in Upstream.